wasm/src/jshelpers.test.ts

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Is there a linter running on all the typescript code that we got?

Reply:
Not currently

Reply:
https://github.com/scaleway/scaleway-cli/issues/3154